### PR TITLE
Add DCAT-AP HVD properties in RDF output if the dataservice is tagged hvd

### PR DESCRIPTION
**Pull request by magopian**
Fixes [#1553](https://github.com/datagouv/data.gouv.fr/issues/1553)

### CODE REVIEW

**Review by maudetes — COMMENTED** [review]
I think I have some questions on applying HVD applicableLegislation based on serving an hvd dataset, since the dataservice may not be sufficient yet.
It may be more cautious to tag dataservices explicitly for now?

Same for HVD category, should they depend HVD datasets being served or be explicit on the dataservice?

I would be interested to discuss this live!

**Comment by magopian** [comment]
I don't have an opinion on the matter, as I don't know enough! I implemented it like that because that's what I understood from [#1533](https://github.com/datagouv/data.gouv.fr/issues/1553)

> La catégorie peut peut-être être déduite des datasets HVD servis par ce dataservice.

I'll change it to only apply it when there's a direct badge on the dataservice, it'll also simplify the code ;)

**Review by bolinocroustibat — APPROVED** [review]
LGTM!

File: udata/core/dataservices/rdf.py [hunk 1]
```diff
@@ -137,8 +141,17 @@ def dataservice_to_rdf(dataservice: Dataservice, graph=None):
     if dataservice.endpoint_description_url:
         d.set(DCAT.endpointDescription, URIRef(dataservice.endpoint_description_url))
 
+    # Add DCAT-AP HVD properties if the dataservice is tagged hvd.
+    # See https://semiceu.github.io/DCAT-AP/releases/2.2.0-hvd/
+    is_hvd = current_app.config["HVD_SUPPORT"] and "hvd" in dataservice.tags
+    if is_hvd:
+        d.add(DCATAP.applicableLegislation, URIRef(HVD_LEGISLATION))
+
     for tag in dataservice.tags:
```
Comment (maudetes) [inline udata/core/dataservices/rdf.py:150]: I think we should look at tags from HVD datasets served by this dataservice instead of its own tags.

**Pull request merged**